res/database/Default/user/cujo.xml

```diff
@@ -0,0 +1,1807 @@
+            </effects>
+            <data genre="2" price="1.0" quality="50" />
+            <availability year_range_from="2010" year_range_to="2012" />
+        </news>
+        <news id="cujo-news-quantar-02" type="2" thread_id="cujo-news-quantar" creator="8827" created_by="Cujo">
+            <title>
+                <de>Sensation - Quantar richtet die Fußball-WM 2022 aus</de>
+                <en>Sensation - Quantar to host the 2022 World Cup</en>
+            </title>
+            <description>
+                <de>Fufa-Generalsekretär Jupp Lauber öffnet den Briefumschlag und liest den Namen des Ausrichters vor. Es ist Quantar.</de>
+                <en>Fufa Secretary General Jupp Lauber opens the envelope and reads out the name of the organizer. It is Quantar.</en>
+            </description>
+            <effects>
+                <effect trigger="happen" type="triggernews" time="2,1,1,13,17" news="cujo-news-quantar-03" />
+            </effects>
+            <data genre="2" price="1.0" quality="90" />
+        </news>
+        <news id="cujo-news-quantar-03" type="2" thread_id="cujo-news-quantar" creator="8827" created_by="Cujo">
+            <title>
+                <de>Wurde die Fufa-WM gekauft?</de>
+                <en>Was the Fufa World Cup bought?</en>
+            </title>
+            <description>
+                <de>War bei der Vergabe der Fufa-WM Korruption im Spiel? SUA, Pajan und Intralien fühlen sich benachteiligt.</de>
+                <en>Was corruption involved in the awarding of the Fufa World Cup? SUA, Pajan and Intralia feel disadvantaged.</en>
+            </description>
+            <effects>
+                <effect trigger="happen" type="triggernews" time="2,1,2,14,18" news="cujo-news-quantar-04" />
+            </effects>
+            <data genre="2" price="1.0" quality="80" />
+        </news>
+        <news id="cujo-news-quantar-04" type="2" thread_id="cujo-news-quantar" creator="8827" created_by="Cujo">
+            <title>
+                <de>Lauber: "Vergabe war völlig in Ordnung."</de>
+                <en>Lauber: "Award was perfectly fine."</en>
+            </title>
+            <description>
+                <de>Fufa-Generalsekretär Jupp Lauber bestreitet alle Korruptionsvorwürfe. "Bei der Fufa gibt es nur ehrliche Menschen."</de>
+                <en>Fufa Secretary General Jupp Lauber denies all allegations of corruption. "There are only honest people at Fufa."</en>
+            </description>
+            <effects>
+                <effect trigger="happen" type="triggernews" time="2,1,2,15,19" news="cujo-news-quantar-05" />
+            </effects>
+            <data genre="2" price="1.0" quality="40" />
+        </news>
+        <news id="cujo-news-quantar-05" type="2" thread_id="cujo-news-quantar" creator="8827" created_by="Cujo">
+            <title>
+                <de>Funktionär aus Trunedad und Topigo gibt Bestechung zu</de>

```

Comment:
Titel nochmal auf "Kürze" prüfen? (Fufa-Funktionär gibt Bestechung zu; Land dann in der Beschreibung)

Reply:
Erledigt

Reply:
Wenn ich es richtig gesehen hatte, steht jetzt "Funktionär aus gibt Bestechung zu"